Makefile

```diff
@@ -331,3 +331,8 @@ check-operatorhub-pr-template:
 local-jaeger-container:
 	@echo "Starting local container with Jaeger. Check http://localhost:16686"
 	@docker run -d --rm -p 16686:16686 -p 6831:6831/udp --name jaeger jaegertracing/all-in-one:1.15 > /dev/null
+
+.PHONY: changelog
+changelog:
+	@echo "Set env variable OAUTH_TOKEN before invoking, https://github.com/settings/tokens/new?description=GitHub%20Changelog%20Generator%20token"
+	@docker run --rm  -v "${PWD}:/app" pavolloffay/gch:latest --oauth-token ${OAUTH_TOKEN} --owner jaegertracing --repo jaeger-operator
```

Comment:
If you externalize those two commands in a shell script, you can do proper env var checking, so that the `docker` command won't be called if the requirements aren't satisfied.

Reply:
Is this really needed? I would like to keep it as simple as possible.

Reply:
Not really needed, it's just being nice to whoever makes a mistake in this step in the future. If the underlying error message is already clear and tells what needs to be done, then it's really not necessary.